include/drivers/usb/usb_dc.h

```diff
@@ -27,6 +27,12 @@
 #define USB_EP_DIR_IN		0x80
 #define USB_EP_DIR_OUT		0x00
 
+/**
+ * USB endpoint attribute type mask.
+ */
+
```

Comment:
I guess this empty line is not necessary.

Reply:
I followed how it was done on the macro above, I'll remove in both places.